src/webview/js/js.js

```diff
@@ -471,9 +471,6 @@ const sendScrollMessage = () => {
     startMessageId: rangeHull.first,
     endMessageId: rangeHull.last,
   });
-  if (!doNotMarkMessagesAsRead) {
-    setMessagesReadAttributes(rangeHull);
-  }
```

Comment:
This looks like the place where the setting actually takes effect; is that right? It seems like this bit should be replaced with something different, instead of removed altogether.